libraries/rush-lib/src/logic/operations/ShellOperationRunnerPlugin.ts

```diff
@@ -71,10 +76,10 @@ function createShellOperations(
 
       const displayName: string = getDisplayName(phase, project);
 
-      if (commandToRun) {
+      if (commandToRun || phase.shellCommand) {
         operation.runner = new ShellOperationRunner({
           buildCacheConfiguration,
-          commandToRun: commandToRun || '',
+          commandToRun: phase.shellCommand || commandToRun || '',
```

Comment:
Since `shellCommand` has been passed to `getScriptToRun`, it may better to leave this block of logic unchanged.